docs/aws/audit/iamcompliance/rules/user_account_with_multiple_access_keys.mdx

```diff
@@ -23,6 +23,226 @@ CISAWS, CBP, HITRUST
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console and open the IAM console at https://console.aws.amazon.com/iam/.
+
+2. In the navigation pane, choose "Users".
+
+3. In the "User Name" list, choose the name of the intended user.
+
+4. In the "User Summary" section, choose the "Security Credentials" tab.
+
+5. Under "Access Keys", check the number of access keys assigned to the user. If there are more than one access keys, then the user account has multiple access keys.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start, you need to install AWS CLI on your local machine. You can do this by using the command `pip install awscli`. After installation, you need to configure it using `aws configure` command. You will be prompted to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. List IAM users: Use the following command to list all the IAM users in your AWS account:
+
+   ```
+   aws iam list-users
+   ```
+
+   This command will return a list of all IAM users in your AWS account.
+
+3. List access keys for each user: For each user, you need to check the number of access keys. You can do this by using the following command:
+
+   ```
+   aws iam list-access-keys --user-name <username>
+   ```
+
+   Replace `<username>` with the name of the IAM user. This command will return a list of all access keys associated with the specified IAM user.
+
+4. Check the number of access keys: If the number of access keys for a user is more than one, then it is a misconfiguration. You can check this by counting the number of access keys returned by the previous command. If the count is more than one, then the user has multiple access keys.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. First, you need to install the AWS SDK for Python (Boto3) if you haven't done so already. You can install it using pip:
+
+```python
+pip install boto3
+```
+
+2. Import the necessary modules and create a session using your AWS credentials:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    region_name='us-west-2'
+)
+```
+
+3. Now, create a client for IAM and list all the users:
+
+```python
+iam = session.client('iam')
+users = iam.list_users()['Users']
+```
+
+4. For each user, list the access keys and check if there are more than one. If there are, print a warning:
+
+```python
+for user in users:
+    access_keys = iam.list_access_keys(UserName=user['UserName'])['AccessKeyMetadata']
+    if len(access_keys) > 1:
+        print(f"Warning: User {user['UserName']} has multiple access keys.")
+```
+
+This script will print a warning for each user that has more than one access key. Note that you need to replace 'YOUR_ACCESS_KEY' and 'YOUR_SECRET_KEY' with your actual AWS credentials.
+</Accordion>
```

Comment:
<h2>Secret Detected: Secret Keyword</h2> <a href="https://console.cloudanix.com/code?t=secrets"> Click here for more details</a>